src/App.svelte

```diff
@@ -180,6 +192,10 @@
     margin-bottom: 0.5rem;
   }
 
+  button.depressed {
```

Comment:
I think "active" or "on" or something like that would better represent what the class does.